frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -19,16 +19,14 @@ import * as monaco from 'monaco-editor';
 import React, { PropsWithChildren } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Link as RouterLink, useLocation } from 'react-router-dom';
-import { useMetadataDisplayStyles } from '.';
-import { LightTooltip } from '..';
 import DetailsViewPluginRenderer from '../../../helpers/renderHelpers';
 import { ApiError } from '../../../lib/k8s/apiProxy';
 import {
   KubeCondition,
   KubeContainer,
   KubeContainerStatus,
   KubeObject,
-  KubeObjectInterface
+  KubeObjectInterface,
```

Comment:
this should not belong here

Reply:
True. I think I copied & paste the entire file due to changing a few things when arranging the commits, and may have removed more than I thought.